pick.lic

```diff
@@ -35,6 +36,12 @@ class Pick
       exit
     end
 
+    if args.stand
```

Comment:
This can be a one liner: 
```
@stand = args.stand
```
If the arg is not nil and not false then will be treated "truthy".

Reply:
Ah, you just taught me this, too. Shall fix.